CHANGES.rst

```diff
@@ -62,7 +62,9 @@ API incompatibility
 
 * ``Matcher`` now requires an additional ``evaluation`` parameter
 * ``Romberg`` removed as an ``NIntegrate[]`` method. It is depcrecated in SciPy and is to be removed by SciPy 1.15.
-* `Definitions.get_ownvalue` now returns a `BaseElement` instead of a `BaseRule` object.
+* The signature of the ``Definition.__init__`` now receives a single dict parameter instead of the severa `*values` parameters.
```

Comment:
severa -> several. 

It is probably time to run through codespell.

Reply:
I did run codespell, and it did not detect it.